db/redis.js

```diff
@@ -53,6 +54,16 @@ async function addDoorTestSensorData(request, response){
     response.status(200).json("OK")
 }
 
+async function addIM21DoorSensorData(locationid, doorSignal){
+    if(doorSignal==SESSIONSTATE_DOOR.CLOSED||doorSignal==SESSIONSTATE_DOOR.OPEN){
```

Comment:
I would be eternally grateful for spaces around the operators `==` and `||` in this line :pray: